Makefile

```
@@ -22,32 +35,48 @@ test: rust-test
+	$(MAKE) -C $(PROJECT_DIR)/telco5g-konflux/scripts/download \
+		download-yamllint \
+		DOWNLOAD_INSTALL_DIR=$(LOCALBIN) \
+		DOWNLOAD_YAMLLINT_VERSION=$(YAMLLINT_VERSION)
 	@echo "Running yamllint on repository YAML files..."
-	yamllint -c $(ROOT_DIR)/.yamllint.yaml .
+	yamllint -c $(PROJECT_DIR)/.yamllint.yaml .
 	@echo "YAML linting completed successfully."
 
 .PHONY: yq
-yq: ## Download yq
+yq: sync-git-submodules $(LOCALBIN)## Download yq
```

checkmake (0.2.2)

[warning] Target body for "yq" exceeds allowed length of 5 (6).

(maxbodylength)